dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/logging/internal/AccessLogger.java

```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
- * 
+ *
```

Comment:
Let's update the copyright range fro 2004, 2022  -> 2004, 2024

Reply:
ok